Cue/app/actions/types.js

```diff
@@ -20,7 +20,8 @@ export type Action =
 	| { type: 'DECK_ALREADY_IN_LIBRARY'}
 	| { type: 'DECK_ADDED_TO_LIBRARY', addedDeck: Deck}
 	| { type: 'CLEAR_INACCESSIBLE_DECKS'}
-;
+	| { type: 'CARD_RATED', change: {uuid: string, userRating: number}}
```

Comment:
Do you mean why I chose to use local changed? Wanted to allow for rating while offline